tensorrt_llm/_torch/models/modeling_utils.py

```
@@ -790,11 +799,40 @@ def forward(
         )
 
     def load_weights(self,
-                     weights: Dict,
+                     weights: dict[str, torch.Tensor],
                      weight_mapper: Optional["BaseWeightMapper"] = None,
```

Ruff (0.16.1)

[error] Undefined name `BaseWeightMapper`

(F821)